### PR TITLE
Feature/lvpytttn 15: group limits

### PR DESCRIPTION
**Pull request by Rocckk**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=896)

## Code reviewers

- [ ] @antooa
- [ ] @HalynaDom
- [ ] @OlexaNdrus
- [ ] @pinkfoxxx
- [ ] @Rocckk
- [ ] @Rostyk1404
- [ ] @Sent1nelDNoir
- [ ] @Svviat
- [ ] @ValentynLiashenko

### Second Level Review

- [ ] @Blackleaves
- [ ] @yevgenbehen
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

### CODE REVIEW

File: iBudget/ibudget/views.py [hunk 1]
```diff
@@ -20,10 +20,11 @@ def post(self, request):# pylint: disable=no-self-use
         try:
             pic = request.FILES['icon']
             if AwsService.check_if_in_bucket(pic):
-                AwsService.upload(pic)
-                pic = str(pic)
-                url = AwsService.get_image_url(pic)
-                return HttpResponse(url)
+                if AwsService.upload(pic):
+                    pic = str(pic)
```
Comment (yevgenbehen) [inline iBudget/ibudget/views.py]: There is no need in this local variable and in general, such kind of variable is a bad smell in a code.

File: iBudget/spending/views.py [hunk 2]
```diff
@@ -74,3 +74,77 @@ def set_spending_limitation_ind(request):
...
+
+def group_limit(request):
+    """the functions finds all the shared spendings associated with particular user and
+    returns them
+    :param request object
+    """
+    if request.method == 'GET':
+        user_id = request.user
+        available_spendings = \
+        SpendingCategories.objects.filter(sharedspendingcategories__group__usersingroups__user_id=
```
Comment (yevgenbehen) [inline iBudget/spending/views.py]: Could you please explain why you select only SpendingCategories for the users that are group admin?

File: iBudget/spending/views.py [hunk 3]
```diff
@@ -74,3 +74,77 @@ def set_spending_limitation_ind(request):
...
+                                          user_id,
+                                          sharedspendingcategories__group__usersingroups__is_admin=
+                                          True).distinct('name')
+        list_of_spensings = []
```
Comment (yevgenbehen) [inline iBudget/spending/views.py]: Typo in `list_of_spensings`

File: iBudget/spending/views.py (hunk 2 above)
Comment (Rocckk) [reply iBudget/spending/views.py]: That's because according to our logic only group admins can set limits for group spendings, not all the group members.

**Pull request merged**